.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -47,7 +45,8 @@ jobs:
           fi
         else
           if [[ -z "${KUBERNETES_VERSION}" ]]; then
-            KUBERNETES_VERSION=$(az aks get-versions -l $AZURE_BUILD_LOCATION --query "orchestrators[?default==\`true\`].orchestratorVersion" -otsv)
+            defaultMajorVersion=$(az aks get-versions -l $AZURE_BUILD_LOCATION --query "values[?isDefault==\`true\`].version" -otsv)
```

Comment:
There is no output when running `az aks get-versions -l eastus --query "values[?isDefault==\`true\`].version" -otsv`

Reply:
Which version of az cli do you use? Maybe you need to do "az upgrade" first.

Reply:
``` 
az version
{
  "azure-cli": "2.50.0",
  "azure-cli-core": "2.50.0",
  "azure-cli-telemetry": "1.0.8",
  "extensions": {
    "fleet": "1.0.0"
  }
}
```

Reply:
Upgrading to 2.56, I got `1.27`. @junjiezhang1997 how do we upgrade azure-cli in test environment?

Reply:
Can we control azure-cli version to avoid that a new cli version breaks current behavior?

Reply:
Use the default azure-cli in test vm. Will fix it if we hit any issue.